ydb/_topic_writer/topic_writer_sync.py

```diff
@@ -56,7 +57,12 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
-        self.close()
+        try:
+            self.close()
+        except ydb.Error:
+            if exc_val:
+                raise exc_val
```

Comment:
maybe `raise exc_val from e` for keep full context?

Reply:
exc_val may contain some context alredy.

In this line I want preserve old exception if it exist.
```exc_val from e``` will write about `e` if cause of `excl_val` and it will difficult to debug.

But the code don't need re-raise exc_val exception at all, i have fixed it, thanks.